VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -283,21 +284,25 @@ public void onSuccess(DiscussionComment comment) {
         if (comment.isEndorsed()) {
             DiscussionThread.ThreadType threadType = discussionThread.getType();
             @StringRes int attributionStringRes;
+            @StringRes int attributionStringResWithoutLabel;
```

Comment:
The 'StringRes' portion is the type, and 'attributionWithoutLabel' is the description. They should be logically partitioned.